### ISSUE TITLE
the resnet implementation is different from tensorflow github

### ISSUE THREAD

**Issue by LeeJayHui** [qubvel/classification_models#57]
Hi, I compare your work with the official implementation of Resnet by tensorflow in [https://github.com/tensorflow/tensorflow/blob/v2.3.0/tensorflow/python/keras/applications/resnet.py](url). There are some notable differences between your work and tensorflow. For example, the downsampling implementation and the batch norm in the input layer. So here is my question, which version of Resnet is more effective? Hope you can help me with this. Thank you.
